zetaclient/chains/evm/signer/gas.go

```diff
@@ -0,0 +1,95 @@
+package signer
+
+import (
+	"math/big"
+
+	"github.com/pkg/errors"
```

Comment:
```suggestion
	"errors"
```
Use errors from std.